_drafts/making-backend-scalable.markdown

```diff
@@ -26,28 +26,27 @@ Therefore we decided to improve our architecture by switching to multi-process m
 
 ## Persistent Memory Model using Redis ##
 
-The in-memory job queue was the barrier to making the system robust. We want to store jobs on an independent system and make it *persistent*
+The in-memory job queue was the barrier to making the system robust. We want to store jobs on an independent system and make it *persistent*.
 
-![Tango with Persistent Memory Model]({{site-url}}/assets/redis2.png)
-_Diagram 2: Tango with Persistent Memory Model_
+![Tango with Persistent Memory Model]({{ site-url }}/assets/redis2.png)
```

Comment:
Spaces vs no spaces between the `{{...}}` is fine, but the hyphen needs to be a dot: `site.url` is a YAML entry called `url` that's accessible from `_config.yml`, so it gets namespaced in the post to `site.`